src/py3s_default_units.py

```diff
@@ -296,6 +298,73 @@ def format(self, output):
                 .format(braces[0], pct_str, braces[1], rem_string, status_string))
 
 
+class PY3Wireless(PY3Unit):
+    """Provide wireless network information.
+
+    Output API:
+        's_status': up, down, none
+        's_SSID'
+        'i_quality'
+
+    Requires:
+        wireless-tools
+    """
+
+    def __init__(self, wlan_id, *args, **kwargs):
+
+        """
+        Args:
+            wlan_id:
+                wireless device name from `ip link`
+                example: wlan0
+        """
+        self.wlan_id=wlan_id
+        super().__init__(*args, requires=['iwconfig'], **kwargs)
+
+    def read(self):
+        # Future: read stats from /proc/net/wireless?
+        # Raw
+        out = check_output(['iwconfig', self.wlan_id]).decode('ascii')
+        line1 = out.split('\n')[0]
+
+        output = {'s_status': "up", "s_SSID": "SSID", "f_quality": 0}
```

Comment:
"SSID" is a silly default. I recommend following the convention of the other unit and introducing some `b_` error flags for when this unit's output is not applicable; i.e. there is no connection.